### DISCUSSION TITLE
mysql-cdc写入hudi,写速度慢的瓶颈

### DISCUSSION THREAD

**Discussion by findczk** [MyLanPangzi/flink-demo#6] [Q&A]
mysql-cdc写入hudi,在快照阶段写入速度较慢,导致第一次完成checkpoint时间长,写入速度为2w/s,每秒2w条记录,每条记录都很小.4个tm,每个tm4g.想知道瓶颈是发生在那个阶段,不甚感激!

**Comment by MyLanPangzi** [comment]
cdc 快照阶段chk是不会成功的.
